Maple2.File.Ingest/Mapper/MapMapper.cs

```diff
@@ -12,8 +12,8 @@ public class MapMapper : TypeMapper<MapMetadata> {
     private readonly MapParser parser;
 
     public MapMapper(M2dReader xmlReader) {
-        spawnParser = new TableParser(xmlReader);
-        parser = new MapParser(xmlReader);
+        spawnParser = new(xmlReader);
```

Comment:
I don't want to use `new()` anywhere. The only scenario that is acceptable to me is for field initialization inline:

```cs
public Type field = new();
```

Because `Type` is obvious.

```cs
Type variable = new(); // This is also acceptable
var variable = new Type(); // but I generally prefer this
```

Reply:
Gotcha